### DISCUSSION TITLE
Is there a way to fix version

### DISCUSSION THREAD

**Discussion by ekersale** [polymind-inc/acmebot#445] [Q&A] [answered]
Hi,

First of all thanks for the awesome tool. 

Is there a way for us to fix the version in order to prevent automatic updates ? 

Thanks for your reply!

**Comment by shibayan** [comment, marked as answer]
Officially, it only provides automatic version upgrades, but you can fix it to any version by building and deploying from source code.

**Reply by ekersale** [reply]
Ok I initially thought that maybe we could modify that variable: 

![image](https://user-images.githubusercontent.com/7691499/149041289-e21fd8db-0f44-4723-9f07-d7df9d861e24.png)

But I get you answer and thank you for it.
